libraries/botbuilder-lg/src/templatesParser.ts

```diff
@@ -209,26 +225,10 @@ export class TemplatesParser {
             }
         }
     }
-
-    private static antlrParseTemplates(text: string, source: string): FileContext {
-        if (!text || text.trim() === '') {
-            return undefined;
-        }
-
-        const input: ANTLRInputStream = new ANTLRInputStream(text);
-        const lexer: LGFileLexer = new LGFileLexer(input);
-        const tokens: CommonTokenStream = new CommonTokenStream(lexer);
-        const parser: LGFileParser = new LGFileParser(tokens);
-        parser.removeErrorListeners();
-        parser.addErrorListener(new ErrorListener(source));
-        parser.buildParseTree = true;
-
-        return parser.file();
-    }
 }
 
 
-class TemplatesTransformer extends AbstractParseTreeVisitor<any> implements LGTemplateParserVisitor<any> {
+export class TemplatesTransformer extends AbstractParseTreeVisitor<any> implements LGTemplateParserVisitor<any> {
```

Comment:
>export [](start = 0, length = 6)

If exporting you should have documentation.

Reply:
#Solved